packages/teams-js/test/public/calendar.spec.ts

```diff
@@ -0,0 +1,275 @@
+import { GlobalVars } from '../../src/internal/globalVars';
+import { FrameContexts } from '../../src/public';
+import { app } from '../../src/public/app';
+import { calendar } from '../../src/public/calendar';
+import { Utils } from '../utils';
+
+describe('calendar', () => {
+  // Use to send a mock message from the app.
```

Comment:
Not really sure you should do anything about this here but it's notable how many times the comments on lines 8, 18, and 23 appear in the codebase.